guides-and-tutorials/running-a-signer/signer-quickstart.md

```diff
@@ -0,0 +1,435 @@
+
+{% tabs %}
+
+{% tab title="Mainnet" %}
+
+```bash
+# Enter the node's datadir
+cd ~/stacks-node/data
+
+# Download the archive
+wget https://archive.hiro.so/mainnet/stacks-blockchain/mainnet-stacks-blockchain-latest.tar.gz
+
+# Decompress the archive
+tar -xvf mainnet-stacks-blockchain-latest.tar.gz
+
+# Remove the archive
+rm mainnet-stacks-blockchain-latest.tar.gz
+```
+
+{% endtab %}
+
+{% tab title="Testnet" %}
+
+```bash
+# Enter the node's datadir
+cd ~/stacks-node/data
+
+# Download the archive
+wget https://archive.hiro.so/testnet/stacks-blockchain/testnet-stacks-blockchain-latest.tar.gz
+
+# Decompress the archive
+tar -xvf testnet-stacks-blockchain-latest.tar.gz
+
+# Remove the archive
+rm testnet-stacks-blockchain-latest.tar.gz
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+### Verify the setup
+
+To ensure the node has been set up correctly, you can run the following commands:
+
+```bash
+# Verify the node's version
+~/stacks-node/stacks-node version
+
+# Output:
+INFO [1729788035.383049] [testnet/stacks-node/src/main.rs:285] [main] stacks-node 3.0.0.0.0 (release/3.0.0.0.0:dd1ebe6, release build, linux [x86_64])
+stacks-node 3.0.0.0.0 (release/3.0.0.0.0:dd1ebe6, release build, linux [x86_64])
+
+# Verify the node's config
+~/stacks-node/stacks-node check-config --config ~/stacks-node/node-config.toml
+
+# Output:
+INFO [1729788064.913032] [testnet/stacks-node/src/main.rs:285] [main] stacks-node 3.0.0.0.0 (release/3.0.0.0.0:dd1ebe6, release build, linux [x86_64])
+INFO [1729788064.913175] [testnet/stacks-node/src/main.rs:318] [main] Loading config at path /home/admin/stacks-node/node-config.toml
+INFO [1729788064.969551] [testnet/stacks-node/src/main.rs:331] [main] Loaded config!
+```
+
+### Start the node
+
+If the outputs of the previous commands are correct, you can proceed and start the node:
+
+```bash
+~/stacks-node/stacks-node start --config ~/stacks-node/node-config.toml
+```
+
+## Step 5 - Generate your signer signature
+
+In order to stack, you'll need your signer signature. The fields required are further explained in the [Generate a signer key signature](https://docs.stacks.co/guides-and-tutorials/stack-stx/stacking-flow#step-2-generate-a-signer-key-signature) guide.
+
+The command to generate a signature looks like this:
+
+```bash
+~/stacks-signer/stacks-signer generate-stacking-signature \
+  --method stack-stx \
+  --max-amount 1000000000000 \
+  --auth-id 195591226970828652622091037492597751808 \
+  --period 12 \
+  --reward-cycle 100 \
+  --pox-address 19tg... \
+  --config ~/stacks-signer/signer-config.toml \
+  --json
+```
+
+The generated JSON can be then copy-pasted directly in the [Lockstacks](https://lockstacks.com) website mentioned in the next step.
+
+## Step 6 - Start stacking
+
+The simplest route is to solo stack. You can do that by using [Lockstacks](https://lockstacks.com). Click on the 'Stack Independently' button and follow the instructions there.
```

Comment:
is this the easiest? since we're already downloading the binaries, we'll have `blockstack-cli` that would be easier IMO, at the expense of a little more text to this doc

Reply:
target audience of this section should be the deciding factor i think.

Reply:
Might not be the easiest to execute, as it would mean navigating to another website and doing the transaction there. It would be the clearest though, as the fields of the signature/transaction are explained on Lockstacks, while doing it through `blockstack-cli` would imply either extra details in this document, or leaving readers without an understanding on what the fields are and why they need them. An extra point is the check on Lockstacks that doesn't allow you to stack unless you have the minimum amount in your wallet.

@kenrogers could you please give your opinion on this as well?

Reply:
If the goal of the quickstart is getting up and running with a signer as quickly as possible, I think we can use the CLI here with a brief description of the commands they need to run and link people to the other methods of stacking if they want more options/details

Reply:
i think i will recant my suggestion here. only because using the cli would require a bit more documentation. it's not as simple apparently as just running the cli with some commands. basically, the pox address needs to be converted to a hash in order to use the cli and that would be out of scope for this document.